DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionTaskInterception.swift

```diff
@@ -61,16 +57,8 @@ public class URLSessionTaskInterception {
         )
     }
 
-    public func register(
-        traceID: TraceID,
-        spanID: SpanID,
-        parentSpanID: SpanID?
-    ) {
-        self.trace = (
-            traceID: traceID,
-            spanID: spanID,
-            parentSpanID: parentSpanID
-        )
+    public func register(trace: TraceContext) {
```

Comment:
We can potentially make `var trace` `public` and drop the `register()` function. I assume it's protocol conformance only?

Reply:
Agreed, tho I'm following the pattern in place for the class.